test/with_dummyserver/test_proxy_poolmanager.py

```diff
@@ -7,7 +7,9 @@
 from dummyserver.testcase import HTTPDummyProxyTestCase, IPv6HTTPDummyProxyTestCase
 from dummyserver.server import (
     DEFAULT_CA, DEFAULT_CA_BAD, get_unreachable_address)
-from .. import TARPIT_HOST
+from .. import (TARPIT_HOST,
```

Comment:
This does not follow the style of any of the other imports. Please either do

```py
from .. import TARPIT_HOST, requires_network
```

or

```py
from .. import (
    TARPIT_HOST, requires_network)
```